modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/LayoutScreen/ObjectLayoutBox.tsx

```diff
@@ -99,8 +99,8 @@ const ObjectLayoutBox: React.FC<IObjectLayoutBoxProps> = ({
 							/>
 						</>
 					}
-					displayCollapseIcon
-					displayDragIcon
+					displayCollapseIcon={displayAddButton}
```

Comment:
@cgmonte @MariMalta Please, rename the `displayAddButton` property to something like `regularBlock`